### ISSUE TITLE
Question : retrieve http status code and headers from useQuery method

### ISSUE THREAD

**Issue by mokabyls** [klis87/redux-requests#374]
Hi,

first of all, many thanks for this awesome library !
I'm facing a problem with React / Redux & Axios Driver. I read many times documentation but i can not find any way to retrieve http status code and headers with getQuery method. The only way i found is to use then method on dispatch. 
Maybe i missing something ?

**Comment by klis87** [comment]
@mokabyls no, actually this is how it is, some people asked for status code and headers and I added this only to dispatch promise.

I wil try to do it in the future so that this will be also compatible with getQuery. For now this works only for errors, as for successful responses only `data` is stored in reducer.

There is a way though, you can use `onRequest` for example to merge response status code or headers with `data`, so then you will get them with `getQuery`

**Comment by mokabyls** [comment]
@klis87 ,

Cool ! I think it’s very useful for many users with rest api. Meanwhile, i will use the promise on dispatch method.
Again, thanks for your work !

**Comment by klis87** [comment]
@mokabyls Thx for the good first issue! Btw in the previous message ofc I meant `onResponse`, not `onRequest`
